### ISSUE TITLE
NVDA is not Announcing Position when Focus is on 'Fully managed' & 'App managed' Tab Items in Browse Mode.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14129]
**Test Environment**:
URL: [Microsoft 365 admin center](https://admin-ignite.microsoft.com/Adminportal/Home#/IntuneDevices)
Browser: Edge Dev 90.0.789.1
OS -Win 10 OS Build 21301.1010
Screen reader: NVDA
 
**Repro Steps**:
1: Open the given URL with above credentials then 'Microsoft M365 admin' page will open.
2: Navigate to 'Devices' menu item and expand it then activate “Active Devices” menu item.
3: Turn on the NVDA, navigate to “App Managed” & "Fully managed" tab items
4: Verified whether NVDA is narrating the position of tab items or not.

**Actual Result**:
NVDA is not announcing position when focus is on 'Fully managed' & 'App managed' tab items in browse mode.

**Expected Result**: 
NVDA should announce position when focus is on 'Fully managed' & 'App managed' tab items in browse mode.

**Attachment**: codepen link - https://codepen.io/testaccount01/pen/jOxrdXR
[Position is not announced in Browse mode (1).zip](https://github.com/nvaccess/nvda/files/9547896/Position.is.not.announced.in.Browse.mode.1.zip)
[Usable_NVDA is not announcing position when focus is on 'Fully managed' & 'App managed' tab items.zip](https://github.com/nvaccess/nvda/files/9547898/Usable_NVDA.is.not.announcing.position.when.focus.is.on.Fully.managed.App.managed.tab.items.zip)

**Comment by rperez030** [comment]
The elements in the example are menu items, not tabs, and they are not part of a list. I wouldn't expect a screen reader to announce position information in this case.

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- This issue appears to be invalid given the above comment.
- The codepen link does not match up with the steps to reproduce, the example URL is not appropriate nor is the code sample. Code samples should be a minimal example to reproduce the issue.
- Please fill out the issue template fully.
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.
